taskcluster/xpi_taskgraph/transforms/release_github.py

```diff
@@ -35,6 +35,8 @@ def build_worker_definition(config, tasks):
             config.params.get("version")
             and config.params.get("xpi_name")
             and config.params.get("build_number")
+            # Don't create Github releases if we're using a fork of `xpi-manifest`
+            and config.params["project"] == "xpi-manifest"
```

Comment:
Should this be `staging-xpi-manifest`? Don't we want these Github releases for `xpi-manifest`?

Reply:
The Github context cuts it off, but there's a `not (...)` around all these conditions.